cloudmarker/test/test_azsqldatabasetdeevent.py

```diff
@@ -0,0 +1,72 @@
+"""Tests for AzSQLDatabaseTDEEvent plugin."""
+
+
+import copy
+import unittest
+
+from cloudmarker.events import azsqldatabasetdeevent
+
+base_record = {
+    'com':  {
+        'record_type': 'database',
```

Comment:
The `com` `record_type` does not appear to be necessary for this test, so I suggest removing this field from the test record.

Reply:
Removed `record_type` from the `com` bucket as it was not used. Thanks for pointing this.